--- a/SunGather/sungather.py
+++ b/SunGather/sungather.py
@@ -138,7 +138,10 @@ def main():
 
     # Load Exports
     exports = configfile.get('exports', [])
-    for export in exports:
+    if exports is None:
+        exports = []
+    
+    for export in exports:
         if export.get('enabled', False):
             export_name = export.get('name')
             logging.info(f"Loading Export: exports/{export_name}")
